testing/simapp/app.go

```diff
@@ -1005,12 +1006,13 @@ func BlockedAddresses() map[string]bool {
 func initParamsKeeper(appCodec codec.BinaryCodec, legacyAmino *codec.LegacyAmino, key, tkey storetypes.StoreKey) paramskeeper.Keeper {
 	paramsKeeper := paramskeeper.NewKeeper(appCodec, legacyAmino, key, tkey)
 
-	// TODO: ibc module subspaces can be removed after migration of params
-	// https://github.com/cosmos/ibc-go/issues/2010
-	paramsKeeper.Subspace(ibctransfertypes.ModuleName)
-	paramsKeeper.Subspace(ibcexported.ModuleName)
-	paramsKeeper.Subspace(icacontrollertypes.SubModuleName)
-	paramsKeeper.Subspace(icahosttypes.SubModuleName)
+	// register the key tables for legacy param subspaces
+	keyTable := ibcclienttypes.ParamKeyTable()
+	keyTable.RegisterParamSet(&ibcconnectiontypes.Params{})
```

Comment:
it looks like you dont really need the returned key table as the `RegisterParamSet` method also manipulates the table itself?  

```// RegisterParamSet registers multiple ParamSetPairs from a ParamSet in a KeyTable.
func (t KeyTable) RegisterParamSet(ps ParamSet) KeyTable {
	for _, psp := range ps.ParamSetPairs() {
		t = t.RegisterType(psp)
	}
	return t
}
```

Reply:
yeah, its operating on a map in `RegisterType` ...seems pointless to return the `KeyTable`

Reply:
yea, i actually found the function signatures on these params keytables to be a bit deceiving, they all just manipulate the table itself but all also return the table :(

Reply:
ah, this is because `KeyTable` is wrapping a map which are basically hidden pointers in go. I'd go with the re-assignment to be on the safe side though I doubt they'll change the internals of `KeyTable` anytime soon.